### PR TITLE
Bug with ie hack selectors

### PR DESCRIPTION
**Pull request by evilebottnawi**
Example
```css
.test {
*display: inline;
*zoom: 1;
}
```

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -14,7 +14,7 @@ var RGBA = /rgba\s*\((\s*(\d+)\s*(,)\s*){3}(\s*(\d?\.\d+)\s*)\)$/i
 module.exports = postcss.plugin("postcss-color-rgba-fallback", function() {
   return function(style) {
     style.eachDecl(function(decl) {
-      if (!decl.value || decl.value.indexOf("rgba") === -1) {
+      if (typeof decl.value !== 'undefined' || decl.value.indexOf("rgba") === -1) {
```
Comment (MoOx) [inline index.js]: How  a value can be undefined ? poke @ai 
Maybe we should do `if (decl.value || String(decl.value).indexOf("rgba") === -1) {`

Comment (evilebottnawi) [inline index.js]: @MoOx may be

**Comment by magsout** [comment]
hum, why kind of bug have you ?

**Comment by magsout** [comment]
If I had your example, I have no error.

**Comment by MoOx** [comment]
Yeah, this is not causing any issue on this playground http://cssnext.io/playground/

**Comment by evilebottnawi** [comment]
@MoOx 
I use gulp with gulp-esnext
```javascript
.pipe(options.environment >= 2 ? $.postcss(processors) : utils.noop())
```
processors - is array postcss plugins

**Comment by MoOx** [comment]
So please double check that it's an issue caused by this plugin.
Some plugin might pass non string value (and they should!)

**Comment by evilebottnawi** [comment]
@MoOx yes, bug in plugin
```javascript
module.exports = postcss.plugin("postcss-color-rgba-fallback", function() {
  return function(style) {
    style.eachDecl(function(decl) {
      // DEBUG
      console.log('Property: ' + decl.prop + '; Value:' + decl.value);
      if (!decl.value || decl.value.indexOf("rgba") === -1) {
        return
      }

      // if previous prop equals current prop
      // if previous prop has hexadecimal value and current prop has rgba() value
      // no need fallback
      if (decl.prev() && decl.prev().prop === decl.prop && decl.value.indexOf("rgba") === decl.prev().value.indexOf("#")) {
        return
      }

      var value = transformRgba(decl.value)
      if (value) {
        decl.cloneBefore({value: value});
      }
    })
  }
});
```
Output:
Property: font-family; Value:sans-serif
Property: -ms-text-size-adjust; Value:100%
Property: -webkit-text-size-adjust; Value:100%
Property: margin; Value:0
Property: display; Value:block
Property: display; Value:-moz-inline-box
Property: display; Value:inline-block
Property: *display; Value:inline
Property: *zoom; Value:1

Bug... Value is integer and have not `indexOf`.

**Comment by evilebottnawi** [comment]
Maybe use toString()?

**Comment by MoOx** [comment]
Like I said, all plugin should play with string. Lot of other plugins do the same test with indexOf. Like too many. Please track this issue https://github.com/postcss/postcss/issues/339 and tell on this one you would like a global fix.

**Comment by evilebottnawi** [comment]
@MoOx right

**Comment by magsout** [comment]
let's close.

**Pull request closed without merging**